plugin/pkg/scheduler/schedulercache/node_info.go

```diff
@@ -39,6 +39,9 @@ type NodeInfo struct {
 	requestedResource *Resource
 	pods              []*api.Pod
 	nonzeroRequest    *Resource
+
+	// FIXME: Comment.
+	generation int64
```

Comment:
do we use generation in other places inside the k8s codebase? I feel version might be more consistent.